Trace.py

```diff
@@ -516,6 +518,53 @@ def dump_surfaces(self, _data, *_args):
 
         self._save_image(img, no_alpha_path, alpha_path)
 
+        depth_path = "command%d--depth.png" % (self.command_count)
```

Comment:
Note: Depending on whether this goes in before or after #42 we'll want to expand this path to capture the draw count and `depth_offset`